### ISSUE TITLE
Duplicate puzzles

### ISSUE THREAD

**Issue by maelic13** [veloce/lichobile#1202]
**I'm submitting a ...**  (keep the relevant one):
- bug report

# Bug report

**Current behavior:**
If I solve puzzles on website and then go to solve puzzles in mobile app, it repeats the same puzzles I just solved on website but without changing rating.

**Expected behavior:**
The puzzles should not be repeated. If I solve puzzles on website and then got o solve in mobile app, I should get new puzzles and rating should change accordingly to my result when solving them.

**Steps to reproduce:**
Solve a few puzzles on website lichess.org and then go into the mobile app logged as the same user. The puzzles you just solved will be offered in the mobile app again and their solving will not change your rating.

**Environment information**

- Samsung Galaxy S10e
- Android 10 latest version for Samsung OneUI 2.1
- v7.0.4

**Comment by landry314** [comment]
Yeah, this is quite annoying. At least I already know the solution and it is quick.

**Comment by bulac3** [comment]
The solution: press "sync and refresh saved puzzles" button
![image](https://user-images.githubusercontent.com/10105366/89134249-47e9bf80-d52c-11ea-95f4-4dc5904a0853.png)

**Comment by maelic13** [comment]
Great! Could the button be pressed automatically when entering the puzzles screen?
